Cpp/odin-views/logout.cpp

```diff
@@ -33,13 +34,13 @@ namespace {
             if ( req.method() == "POST" ) {
                 auto logout_claim =
                     req.headers()["__jwt"].subvalue(odin::c_jwt_logout_claim.value());
-                if ( logout_claim ) {
-                    fostlib::pg::connection cnx{fostgres::connection(config, req)};
-                    fostlib::json row;
-                    fostlib::insert(row, "identity_id", req.headers()["__user"].value());
-                    fostlib::insert(row, "reference", req.headers()["__odin_reference"].value());
-                    fostlib::insert(row, "source_address", req.headers()["__remote_addr"].value());
-                    cnx.insert("odin.logout_ledger", row);
+                fostlib::pg::connection cnx{fostgres::connection(config, req)};
+                if ( odin::does_module_enabled(cnx, "opts/logout") && logout_claim ) {
```

Comment:
Why do we need this new check here? Is it not enough to see if the logout claim is in the JWT?